client/src/main/java/gui/data/Field.java

```diff
@@ -0,0 +1,47 @@
+package gui.data;
+
+import model.Coordinates;
+
+import java.util.Map;
+import java.util.Objects;
+
+public class Field {
+    private Map<Coordinates, FieldState> coordinatesState;
+
+    private Field(Map<Coordinates, FieldState> coordinatesState) {
+        this.coordinatesState = coordinatesState;
+    }
+
+    public static Field create(Map<Coordinates, FieldState> coordinatesState) {
```

Comment:
ofCoordinateState

Reply:
Will leave it like it is. Too long name of factory method in my opinion.